src/webapi/window.rs

```diff
@@ -86,4 +88,17 @@ impl Window {
             ).into_reference_unchecked()
         }
     }
+
+    /// Sets a callback in requestAnimationFrame to call on next frame
+    /// Callback accepts an f64 as an argument
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)
+    pub fn request_animation_frame< F: FnMut(f64) + 'static>( &self, callback: F) -> RequestAnimationFrameHandle {
+        let id : u32 = js!{
+            var callback = @{callback};
+            var request = requestAnimationFrame(callback);
+            return request;
+        }.try_into().unwrap_or(0);
```

Comment:
Why the `unwrap_or`? Shouldn't it be simply `unwrap`?

Reply:
I mostly just did unwrap_or out of habit, changed it.